data-pipeline/terraform/data.tf

```diff
@@ -13,32 +8,7 @@ data "azurerm_log_analytics_workspace" "application-insights-workspace" {
   resource_group_name = "${var.environment-prefix}-ebis-core"
 }
 
-data "azurerm_container_registry" "acr" {
-  name                = "${var.environment-prefix}acr"
-  resource_group_name = "${var.environment-prefix}-ebis-core"
-}
-
 data "azurerm_storage_account" "main" {
```

Comment:
Hmmm…as the resulting `primary_connection_string` is sensitive, this should probably come out too.